### PR TITLE
All meal totals

### PR DESCRIPTION
**Pull request by mikeheft**
@snayrouz Made the delete from meals more fluid and added some styling. Basically done. I'm going to take a break for lunch and the winter fest thing here and I'll look at this one console log error in the foods page. It doesn't affect the functionality at all though

### CODE REVIEW

**Comment by snayrouz** [comment]
Thanks Mike I’m going to play around with everything tomorrow and do some Sammy styles

**Comment by mikeheft** [comment]
@snayrouz conflicts resolved. 😄

**Comment by snayrouz** [comment]
Sweet thanks!

File: index.html [hunk 1]
```diff
@@ -38,8 +38,8 @@ <h4>Search by Name</h4>
         <thead>
           <tr>
             <th class="hidden"></th>
-            <th>Name</th>
-            <th>Calories</th>
+            <th id='name'>Name</th>
+            <th id='cal' class='cal-header'><button class='sort-cal'>Calories</button></th>
```
Comment (snayrouz) [inline index.html:42]: Nice

File: lib/meals.js [hunk 2]
```diff
@@ -78,32 +81,43 @@ const addFoodToMeal = () => {
...
-    $.get(`${api}/meals/${mealId}/foods`, data => {
-      $('input[type=checkbox]').prop('checked',false);
-      $(`table#${mealId} tr:not(:first-child)`).remove()
-      mealFoods(data)
+      }).then(reGetMeal(mealId))
     })
   })
 }
 
+const reGetMeal = (mealId) => {
```
Comment (snayrouz) [inline lib/meals.js:89]: The regret regret

**Review by snayrouz — APPROVED** [review]

**Pull request merged**